topcoffea/modules/objects.py

```diff
@@ -50,7 +50,7 @@ def isCleanJet(jets, electrons, muons, taus, drmin=0.4):
   return (jetEleMask & jetMuMask & jetTauMask)
 
 def isTightJet(pt, eta, jet_id, neHEF, neEmEF, chHEF, chEmEF, nConstituents, jetPtCut=30.0):
-    mask = (pt>jetPtCut) & (abs(eta)<2.4)# & ((jet_id&6)==6)
+    mask = (pt>jetPtCut) & (abs(eta)<2.4) & ((jet_id&6)==6)
```

Comment:
Will this break with data, or is `jet_id` set by the jet tagging algorithm?

Reply:
jet_id is a reco variable, present in data